package.json

```diff
@@ -38,7 +38,7 @@
     "concurrently": "^5.0.0",
     "css-loader": "^3.2.1",
     "cypress": "^3.8.2",
-    "fetch-mock": "^8.0.0",
+    "fetch-mock": "^9.9.0",
```

Comment:
also upgrades fetch-mock. Totally unrelated to the rest of the PR but I needed to test out a new release in a few projects and treecreeper is  a good 'n' complex one. May as well keep the upgrade now it's done